imod/tests/test_mf6/test_multimodel/test_mf6_partitioning_unstructured.py

```diff
@@ -216,6 +290,54 @@ def test_partitioning_unstructured_with_vpt_cells(
     )
 
 
+@pytest.mark.usefixtures("circle_model")
+@parametrize_with_cases("partition_array", cases=PartitionArrayCases)
+@parametrize_with_cases("hfb", cases=HorizontalFlowBarrierCases)
+def test_partitioning_unstructured_hfb(
+    tmp_path: Path,
+    circle_model: Modflow6Simulation,
+    partition_array: xu.UgridDataArray,
+    hfb: imod.mf6.HorizontalFlowBarrierBase,
+):
+    simulation = circle_model
+    # increase the recharge to make the head gradient more pronounced
```

Comment:
Please start your comments with a captial. They should be as close to proper english as possible.

https://peps.python.org/pep-0008/#comments

> Comments should be complete sentences. The first word should be capitalized, unless it is an identifier that begins with a lower case letter (never alter the case of identifiers!).

Reply:
Thanks, done!